oeps/oep-0011-bp-FED-technology.rst

```diff
@@ -226,14 +258,23 @@ Rejected Alternatives
 .. _Babel: https://babeljs.io/
 .. _edX ESLint Config: https://github.com/edx/eslint-config-edx/tree/master/packages/eslint-config-edx
 .. _edX ESLint Config for ES5: https://github.com/edx/eslint-config-edx/tree/master/packages/eslint-config-edx-es5
+.. _edx Frontend Auth Client: https://github.com/edx/frontend-auth
+.. _enzyme: https://airbnb.io/enzyme/
 .. _ESLint configuration cascading: http://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy
 .. _ES2015 Modules: http://www.ecma-international.org/ecma-262/6.0/#sec-imports
 .. _ES2017: https://tc39.github.io/ecma262/
+.. _Fetch: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
+.. _Flow: https://flow.org/
+.. _Greenkeeper: https://greenkeeper.io/
+.. _Jest: https://jestjs.io/
 .. _JSX: https://facebook.github.io/react/docs/introducing-jsx.html
 .. _Modernizing the edX front end stack: https://openedx.atlassian.net/wiki/display/FEDX/Modernizing+the+edX+front+end+stack
```

Comment:
**Question:** Is there any value in linking back to the `Modernizing edX FE` Confluence page (which is now archived)?  Our rationale and decisions should be self-contained in this OEP, right?

Reply:
Agreed. Removing.